### ISSUE TITLE
ala-cas jsp uses hardcoded URLs to http://auth.ala.org.au

### ISSUE THREAD

**Issue by mbohun** [AtlasOfLivingAustralia/ala-cas#9]
```
bash-3.2$ pwd
/Users/hor22n/src/ala-cas_mbohun.git
bash-3.2$ find ./ -name "*.jsp" | xargs grep http | grep auth
.//src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp:                <p><spring:message code="screen.welcome.notRegistered" arguments="http://auth.ala.org.au/emmet/selfRegister.html"/></p>
.//src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp:                <p><spring:message code="screen.welcome.forgottenpassword" arguments="http://auth.ala.org.au/emmet/resetMyPassword.html"/></p>
```
make this configureable:
- store and read the URLs from a `.properties` file
- when installing `ala-cas` with `ala-install` ansible scripts adjust the URLs to point to the host/IP `ala-cas` is being installed on (to allow for a smooth automated/deployment)

**Comment by djtfmartin** [comment]
Follow the approach in biocache-service which is reading the external config file in

/data/{{appName}}/config/{{appName}}-config.properties

**Comment by djtfmartin** [comment]
See

https://github.com/AtlasOfLivingAustralia/biocache-service/blob/master/src/main/webapp/WEB-INF/mainDispatcher-servlet.xml#L29

and copy this approach

**Comment by m-r-c** [comment]
Fixed in version 1.1.

Waiting on pull request #11
